include/mc_tasks/MetaTask.h

```diff
@@ -247,6 +247,19 @@ struct MC_SOLVER_DLLAPI MetaTask
     t.removeFromGUI(gui);
   }
 
+  void ensureHasJoints(const mc_rbdyn::Robot & robot,
```

Comment:
`inline static void ...`

(this does not require an instance of a task to work)